src/DayColumn.js

```diff
@@ -230,7 +233,12 @@ class DayColumn extends React.Component {
               [isRtl ? 'right' : 'left']: `${Math.max(0, xOffset)}%`,
               width: `${width}%`,
             }}
-            title={(typeof label === 'string' ? label + ': ' : '') + title}
+            {...(tooltip
+              ? {
+                  title:
+                    (typeof label === 'string' ? label + ': ' : '') + tooltip,
+                }
+              : {})}
```

Comment:
lets make this a bit less ugly please :P i'd just set title to `undefined` in the one case that will be fine